### PR TITLE
fix: signature issues

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [] 

### Description
This PR makes the following changes:
- Update the text font size to match palette
- Tapping on signature was slow, I made it faster
- Make the signature mandatory

https://github.com/artsy/eigen/assets/11945712/857ffdd4-fe7d-4eae-a42f-5ef693cb32b5

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

#nochangelog

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b581cd169a9eabd7e489f7ef760e0ed614944103
</p>

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkPurchaseHistory.tsx [hunk 1]
```diff
@@ -17,6 +18,13 @@ export const PROVENANCE_LIST = [
 
 export const SubmitArtworkPurchaseHistory = () => {
   const { setFieldValue, values } = useFormikContext<ArtworkDetailsFormModel>()
+  const [isSigned, setIsSigned] = useState(values.signature)
+
+  useEffect(() => {
+    if (isSigned !== null && isSigned !== undefined) {
+      setFieldValue("signature", isSigned)
+    }
+  }, [isSigned, setFieldValue])
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkPurchaseHistory.tsx:27]: A comment would be great about why we are not using Formik directly.

**Review by olerichter00 — APPROVED** [review]

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkPurchaseHistory.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkPurchaseHistory.tsx:27]: good idea

Comment (olerichter00) [reply src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkPurchaseHistory.tsx:27]: Thanks!

**Pull request merged**